core/src/test/scala/unit/kafka/utils/CoreUtilsTest.scala

```diff
@@ -223,7 +204,7 @@ class CoreUtilsTest extends Logging {
     val map = new ConcurrentHashMap[Int, AtomicInteger]().asScala
     implicit val executionContext = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(nThreads))
```

Comment:
As you are cleaning up some minor problems, feel free to type this `implicit val` as this is the recommended way:
```suggestion
    implicit val executionContext: ExecutionContext = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(nThreads))
```

Reply:
I've add to use `ExecutionContextExecutorService` instead of `ExecutionContext` because we call `shutdownNow()` below.

Reply:
Looks good!